tests/lib/snaps/test-devpts/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-devpts
```

Comment:
Not a blocker but I think we are trying to name those `test-snapd-...`

Reply:
Done